src/frontend/src/views/AgreeTerms/index.tsx

```diff
@@ -36,91 +42,96 @@ export default function AgreeTerms(): JSX.Element {
   }
 
   return (
-    <Container>
```

Comment:
This looks like a huge change, but the only thing I added is `<Head><title>...</title></Head>`